modules/ROOT/pages/file-name-pattern.adoc

```diff
@@ -1,8 +1,8 @@
 = File Name Patterns for Send Endpoints
 
-Partner Manager enables you to configure the file name patterns to use on the target files delivered to SFTP and AS2 Send endpoints, through the use of property placeholders.
+Using property placeholders, Partner Manager enables you to configure the file name patterns for target files delivered to SFTP and AS2 send endpoints.
 
-When the file name pattern is configured with property placeholders, specified within `${ }`, Partner Manager replaces the property dynamically at runtime. Any string that is not specified within the placeholder is used as static text in the file name.
+When file name pattern fields are configured with property placeholders (specified within `${ }`), Partner Manager dynamically populates the file name with updated information at runtime. Any file name property field that is _not_ specified within the placeholder is static and is not updated at runtime.
```

Comment:
Avoid formatting to indicate emphasis. I think simply using "not" is enough.

Reply:
Done